Data/DbInitializer.cs

```diff
@@ -0,0 +1,35 @@
+using DiAnterExpress.Models;
+using System;
+using System.Linq;
+
+namespace DiAnterExpress.Data
+{
+    public static class DbInitializer
+    {
+        public static void Initialize(ApplicationDbContext context)
+        {
+            context.Database.EnsureCreated();
+
+            if (context.Branches.Any())
+            {
+                return;
+            }
```

Comment:
ini logic nya agak kurang tepat.

Misalkan ada kondisi dimana table `Branch` udah ada isinya, sedangkan table lain masih kosong, `DbInitializer`nya nggak bakal jalan.

Mungkin bisa diganti jadi :
```suggestion
            if (!context.Branches.Any())
            {
                var branches = new Branch[]
                {
                    new Branch{Name ="Di Anter Express Pondok Indah",Address="Jln. Pondok Indah Ruko No.19",City="Jakarta",Phone=0812345679},
                }
                //dan seterusnya sampe context.SaveChanges();
                
            }
```

Reply:
Oke sudah mas